browser/ui/webui/settings/brave_appearance_handler.cc

```diff
@@ -108,6 +113,12 @@ void BraveAppearanceHandler::RegisterMessages() {
       "getIsBitcoinDotComSupported",
       base::BindRepeating(&BraveAppearanceHandler::GetIsBitcoinDotComSupported,
                           base::Unretained(this)));
+#if BUILDFLAG(CRYPTO_DOT_COM_ENABLED)
+  web_ui()->RegisterMessageCallback(
+      "getIsCryptoDotComSupported",
```

Comment:
I think this API needs to be available when CRYPTO_DOG_COM_ENABLED is false.

Reply:
yep, fixed